File: semestre-1/algoritmos/MR-Alg-01/c/MR-Alg-01-Ex-05.c
#include <stdio.h>

void main() {
   int tipo1, tipo2;

   printf("Vasilhames de 1 litro ou menos: ");
   scanf("%i", &tipo1);

   printf("Vasilhames de mais de 1 litro: ");
   scanf("%i", &tipo2);

   float credito = tipo1 * .10 + tipo2 * .25;
   printf("Crédito referente ao retorno dos vasilhames: R$ %.2f\n", credito);
}

File: semestre-1/algoritmos/MR-Alg-01/.py/MR-Alg-01-Ex-05.py
tipo1 = int(input("Vasilhames de 1 litro ou menos: "))
tipo2 = int(input("Vasilhames de mais de 1 litro: "))

credito = tipo1 * .10 + tipo2 * .25
print(f"Crédito referente ao retorno dos vasilhames: R$ {credito:.2f}")